CMakeLists.txt

```diff
@@ -41,15 +41,38 @@ FIND_PACKAGE( DD4hep REQUIRED )
 FIND_PACKAGE( ROOT REQUIRED )
 FIND_PACKAGE( MarlinTrk REQUIRED )
 
+OPTION( USE_SDHCALCONTENT "Use SDHCALContent" OFF )
+OPTION( USE_APRILCONTENT "Use APRILContent" OFF )
 
-FOREACH( pkg PandoraSDK Marlin MarlinUtil  LCContent DD4hep ROOT MarlinTrk )
+FOREACH( pkg PandoraSDK Marlin MarlinUtil LCContent DD4hep ROOT MarlinTrk )
     IF( ${pkg}_FOUND )
         INCLUDE_DIRECTORIES( SYSTEM ${${pkg}_INCLUDE_DIRS} )
         LINK_LIBRARIES( ${${pkg}_LIBRARIES} )
         ADD_DEFINITIONS ( ${${pkg}_DEFINITIONS} )
     ENDIF()
 ENDFOREACH()
 
+IF( USE_SDHCALCONTENT )
+    FIND_PACKAGE( SDHCALContent REQUIRED )
+    INCLUDE_DIRECTORIES( SYSTEM ${SDHCALContent_INCLUDE_DIRS} )
+    LINK_LIBRARIES( ${SDHCALContent_LIBRARIES} )
+    ADD_DEFINITIONS ( ${SDHCALContent_DEFINITIONS} )
+    ADD_DEFINITIONS( "-DSDHCALCONTENT" )
+ENDIF()
+
+IF( USE_APRILCONTENT )
+    SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+    FIND_PACKAGE( APRILContent REQUIRED )
+    FIND_PACKAGE( mlpack REQUIRED )
+    INCLUDE_DIRECTORIES( SYSTEM ${APRILContent_INCLUDE_DIRS} )
+    LINK_LIBRARIES( ${APRILContent_LIBRARIES} )
+    ADD_DEFINITIONS ( ${APRILContent_DEFINITIONS} )
```

Comment:
```suggestion
```
`APRILContent` doesn't seem to export any definitions(?): https://github.com/SDHCAL/APRILContent/blob/current/cmake/APRILContentConfig.cmake.in

Reply:
Same as for SDHCALContent, I also deleted it